src/Types/User.cpp

```diff
@@ -0,0 +1,9 @@
+/** ----------------------------------------------------------------------------
```

Comment:
this file should be removed, since it's empty

Reply:
done